app/objects/c_adversary.py

```diff
@@ -11,17 +11,15 @@ def unique(self):
 
     @property
     def display(self):
-        phases = dict()
-        for k, v in self.phases.items():
-            phases[k] = [val.display for val in v]
-        return dict(adversary_id=self.adversary_id, name=self.name, description=self.description, phases=phases)
+        return dict(adversary_id=self.adversary_id, name=self.name, description=self.description,
+                    listing=[x.ability_id for x in self.atomic_ordering])
```

Comment:
listing or just `abilities` or `atomic`? We are jumping between language across objects.  Lets select on and standardize